### PR TITLE
Updated GraphQL queries

### PR DESCRIPTION
**Pull request by NimSum**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

```Updated the queries that Matt has already implemented on the backend```

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling change (non-breaking change which adds styling)
- [ ] Other, Please explain:

## Has This Been Tested?
- [ ] No
- [x] Yes
```BUT the clientQuery file still needs to get tested, in progress!```

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

```Your message here```

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have no linter errors in my code
- [x] I have run testing on my code to ensure they are all passing
- [x] I will comment on my code, particularly in hard-to-understand areas

# Message for reviewer:

```Cleaned up the App component, I've tested the graphQL queries via the methods we are using in the queries file. Feel free to try it yourself so that you have better familiarity with it!```

### CODE REVIEW

File: src/utils/queries/queries.js [hunk 1]
```diff
@@ -2,8 +2,11 @@ import { gqlQuery, gqlMutate } from './clientQuery';
 import * as queryTypes from './queryTypes';
 
 const queries = {
```
Comment (tiffanybacher) [inline src/utils/queries/queries.js]: Should this be called queries or something else?

File: src/utils/queries/clientQuery.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 import ApolloClient from 'apollo-boost';
 
 const client = new ApolloClient({
-  uri: 'https://cors-anywhere.herokuapp.com/https://pure-castle-14648.herokuapp.com/graphql'
+  uri: 'https://cors-anywhere.herokuapp.com/https://pure-castle-14648.herokuapp.com/graphql',
+  request: (operation) => {
+    const token = localStorage.getItem('token');
```
Comment (ehk9000) [inline src/utils/queries/clientQuery.js]: who knew local storage would come in handy again? 😂

Comment (tiffanybacher) [reply src/utils/queries/clientQuery.js]: 😂

**Comment by tiffanybacher** [comment]
Thanks for updating the naming on all the gql and Appllo pieces. Makes following the requests a little easier!

**Pull request merged**